packages/lib/src/vis/tiles/utils.ts

```diff
@@ -10,15 +11,15 @@ export function getTileOffsets(
 ): Vector2[] {
   const { width, height } = tileSize;
 
-  const [xOrigin, xEnd] = xDomain;
-  const [yOrigin, yEnd] = yDomain;
+  const [xMin, xMax] = isDescending(xDomain) ? xDomain.reverse() : xDomain;
+  const [yMin, yMax] = isDescending(yDomain) ? yDomain.reverse() : yDomain;
```

Comment:
Should you make a copy here according to [your comment](https://github.com/silx-kit/h5web/pull/1052/files#r841662024) ?